classes/ActionScheduler_Admin_Actions_Rest_Controller.php

```diff
@@ -206,7 +206,8 @@ public function get_actions( $request ) {
 		];
 
 		try {
-			$timezone = new DateTimeZone( get_option( 'timezone_string' ) );
+			$timezone_string = get_option( 'timezone_string' );
```

Comment:
What do you think of using `wp_timezone_string()` to get this value? It occurs that if we want to use the configured timezone, that would also cover a scenario where a GMT offset such as `+04:00` is in effect (since, when in use, GMT offsets are stored under a different key).

Reply:
Great suggestion. Updated in https://github.com/woocommerce/action-scheduler-admin/pull/47/commits/afe2ab6948d3f011b6bfdb202474a2ff5cb14479.